libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3606,7 +3613,7 @@ void GCS_MAVLINK::handle_statustext(const mavlink_message_t &msg) const
     }
 
     memcpy(&text[offset], packet.text, MAVLINK_MSG_STATUSTEXT_FIELD_TEXT_LEN);
-
+    send_text(MAV_SEVERITY_INFO, "%s", text); 
```

Comment:
Instant traffic reflection attack - if you have two ArduPilot instances that can see each other messages will never leave the system except via dropped messages or resource starvation.  The fun really starts if you have thee instances.

Reply:
> Instant traffic reflection attack - if you have two ArduPilot instances that can see each other messages will never leave the system except via dropped messages or resource starvation. The fun really starts if you have thee instances.

Is this true even if different ardupilot instances have different SYSID_THISMAV?